Empty javadoc results in IndexOutOfBoundsException in JavadocParser

See https://github.com/revelc/impsort-maven-plugin/issues/16

Parsing of empty javadoc, eg:

    /**
    */
    public class SomeClass {
        List something = new ArrayList();
    }

Fails with IndexOutOfBoundsException

    Caused by: java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
    at java.util.ArrayList.rangeCheck (ArrayList.java:653)
    at java.util.ArrayList.get (ArrayList.java:429)
    at com.github.javaparser.JavadocParser.cleanLines (JavadocParser.java:124)
    at com.github.javaparser.JavadocParser.parse (JavadocParser.java:50)

--- a/javaparser-core-testing/src/test/java/com/github/javaparser/JavadocParserTest.java
+++ b/javaparser-core-testing/src/test/java/com/github/javaparser/JavadocParserTest.java
@@ -39,6 +39,12 @@ class JavadocParserTest {
     }
 
     @Test
+    void parseEmptySingleLine() {
+        Assertions.assertEquals(new Javadoc(JavadocDescription.parseText("")),
+                JavadocParser.parse(EOL));
+    }
+
+    @Test
     void parseSingleLineWithSpacing() {
         assertEquals(new Javadoc(JavadocDescription.parseText("The line number of the first character of this Token.")),
                 JavadocParser.parse(" The line number of the first character of this Token. "));

--- a/javaparser-core/src/main/java/com/github/javaparser/JavadocParser.java
+++ b/javaparser-core/src/main/java/com/github/javaparser/JavadocParser.java
@@ -101,6 +101,10 @@ class JavadocParser {
 
     private static List<String> cleanLines(String content) {
         String[] lines = content.split(EOL);
+        if (lines.length == 0) {
+            return Collections.emptyList();
+        }
+
         List<String> cleanedLines = Arrays.stream(lines).map(l -> {
             int asteriskIndex = startsWithAsterisk(l);
             if (asteriskIndex == -1) {

